public/jsx/app.jsx

```diff
@@ -6,6 +6,7 @@ var Route = Router.Route;
 var RouteHandler = Router.RouteHandler;
 
 var CardDetails = require('views/card-details');
+var Complete = require('views/complete');
```

Comment:
nit: this makes sense as a URL but it's a little vague on its own. Maybe ``CompletePay`` and ``views/complete-pay`` ?